AWS/cloudformation-templates/heroku-app-and-cloudfront.json

```diff
@@ -145,7 +145,37 @@
           }
         }
       }
+    },
+    "DeployUser": {
+      "Type": "AWS::IAM::User",
+      "Condition": "CreateDeployAccount",
+      "Properties": {
+        "Path": "/deploy/",
+        "Policies": [{
+          "PolicyName": {
+            "Fn::Join": ["", ["S3-RW-", {"Ref":"S3Bucket"}]]
```

Comment:
Could you change the name of this policy to something more descriptive of what it's doing?